components/gallery/GalleryItemTabsPanel/GalleryItemOffers.vue

```diff
@@ -92,7 +95,7 @@ const { $i18n, $consola } = useNuxtApp()
 
 const { apiInstance } = useApi()
 const { urlPrefix, tokenId, assets } = usePrefix()
-const { decimals } = useChain()
+const { decimals, unit } = useChain()
```

Comment:
imo maybe the `useChain` function need update? may do it in another pr
cc @roiLeo

Reply:
What do you mean by "update"? This change won't work as it show BSX where it should be KSM
![Screenshot 2023-04-28 at 08-21-45 331660682-7 KodaDot - NFT Market Explorer](https://user-images.githubusercontent.com/9987732/235070174-89752c04-2d82-413c-a806-fa42f1ecd691.png)

Reply:
> What do you mean by "update"? This change won't work as it show BSX where it should be KSM 

sry for my description, i mean is it a good way to change the `useChain` function directly to display `KSM` by default currently?

Reply:
> sry for my description, i mean is it a good way to change the `useChain` function directly to display `KSM` by default currently?

We want to avoid to hardcode KSM value so I won't change this composable, the goal is to display same way as we display price currency

Reply:
fixed now @roiLeo @floyd-li used chainSymbol variable

Reply:
@prachi00 please check deepsource issue

Reply:
> > sry for my description, i mean is it a good way to change the `useChain` function directly to display `KSM` by default currently?
> 
> We want to avoid to hardcode KSM value so I won't change this composable, the goal is to display same way as we display price currency

aya. got it. 
just a little question, what's the difference between `unit` and `chainSymbol` in `useChain` 🤔️?

Reply:
Good Q. can't really give a proper answer but from what I know/see `unit` is the chain currency while `chainSymbol` is the asset used

Reply:
Correctly 💚. 
I should make you special koda tshirt

unit is the currency of chain:
Kusama has KSM
Basilisk has BSX

but for example we do we KSM token on Basilisk chain.